package.json

```diff
@@ -26,5 +29,9 @@
     "lint": "eslint --ext js --ext jsx src test; exit 0",
     "lint:fix": "eslint --fix --ext js --ext jsx src test; exit 0"
   },
-  "homepage": "."
+  "homepage": ".",
+  "devDependencies": {
+    "react-test-renderer": "^16.4.1",
+    "redux-mock-store": "^1.5.3"
```

Comment:
hmmm... we have these currently as regular dependencies, but I think you may be right that they should be dev dependencies.